aladdin/bash/container/create-namespace/create-namespace

```diff
@@ -3,14 +3,18 @@ set -eu -o pipefail
 
 function create_namespace {
     kubectl create namespace $1
+    kubectl label --overwrite namespace $1 owner=$2
```

Comment:
maybe ignore if there's an error here? (like invalid characters in value, or something of that nature)
```suggestion
    kubectl label --overwrite namespace $1 owner=$2 ||:
```
since it's a non-critical piece of functionality (I think)